sagemaker-debugger/tensorflow2/tensorflow2_keras_to_estimator/tf2-keras-model-to-estimator.ipynb

```diff
@@ -0,0 +1,407 @@
+{
```

Comment:
<p>do we need this ?</p><p>By default it should pick /opt/ml/output/tensors</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1298/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
The training script has out_dir as mandatory argument.